### PR TITLE
Feature/rn65.1

### PR DESCRIPTION
**Pull request by mlobac**
## What does this PR do?
Update react-native version to 65.1

- [+] Checked on iOS
- [+] Checked on Android

### CODE REVIEW

File: src/navigators/RootNavigator.tsx [hunk 1]
```diff
@@ -72,6 +74,8 @@ export const RootNavigator: FC<Props> = ({
   shouldRenderNotification,
   userVersion,
 }) => {
+  useFlipper(navigationRef);
```
Comment (panradadev) [inline src/navigators/RootNavigator.tsx:77]: It will not touch production that ?

Comment (mlobac) [reply src/navigators/RootNavigator.tsx:77]: No, it doesn't work in production

**Review by panradadev — APPROVED** [review]

File: android/gradle.properties [hunk 2]
```diff
@@ -23,4 +23,4 @@ org.gradle.daemon=true
 org.gradle.jvmargs=-Xmx2560m
 
 # Version of flipper SDK to use with React Native
-FLIPPER_VERSION=0.99.0
\ No newline at end of file
+FLIPPER_VERSION=0.93.0
```
Comment (aleparty) [inline android/gradle.properties:26]: might be a stupid question, but downgraded version is correct one for upgraded RN?

Comment (mlobac) [reply android/gradle.properties:26]: I mistakely upgraded to higher version then required and it didn't work ;) So this version - correct

**Review by aleparty — APPROVED** [review]

File: android/gradle.properties (hunk 2 above)
Comment (aleparty) [reply android/gradle.properties:26]: Ok

**Pull request merged**